compiler-rt/lib/sanitizer_common/sanitizer_stacktrace_printer.cpp

```diff
@@ -322,11 +327,12 @@ void FormattedStackTracePrinter::RenderData(InternalScopedString *buffer,
   }
 }
 
-#endif  // !SANITIZER_SYMBOLIZER_MARKUP
+#endif  // !SANITIZER_SYMBOLIZER_MARKUP_FUCHSIA
 
-void FormattedStackTracePrinter::RenderSourceLocation(
-    InternalScopedString *buffer, const char *file, int line, int column,
-    bool vs_style, const char *strip_path_prefix) {
+void StackTracePrinter::RenderSourceLocation(InternalScopedString *buffer,
+                                             const char *file, int line,
+                                             int column, bool vs_style,
+                                             const char *strip_path_prefix) {
```

Comment:
Since this and `RenderModuleLocation` don't seem to be changed, it might be simpler/easier to read this patch if these changes were in a separate NFC patch.

Reply:
The change is that this and `RenderModuleLocation` have the same implementation for the `FormattedStackTracePrinter` and the `MarkupStackTracePrinter` before introducing the Markup specific code, this change made no much sense.